From: Marco Scardovi <marco@scardovi.com>
Date: Sat, 1 Jan 2022 15:20:45 +0100
Subject: make HP 15s-eq0xxx changes specific to sku9MG38EA#ABZ

Signed-Off-By: Marco Scardovi <marco@scardovi.com>
(cherry picked from commit 7bd3d6e35a6de8b1bf93e2fae28a64f0c7ffd2ac)
---
 hwdb.d/60-keyboard.hwdb | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 2c1cdeb..b614a22 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -561,8 +561,8 @@ evdev:atkbd:dmi:bvn*:bvr*:svnHP*:pnHPElitex21013G3:*
  KEYBOARD_KEY_92=brightnessdown
  KEYBOARD_KEY_97=brightnessup
 
-# HP Laptop15s-eq0xxx
-evdev:atkbd:dmi:bvn*:bvr*:svnHP*:pnHPLaptop15s-eq0*:*
+# HP Laptop 15s-eq0023nl
+evdev:atkbd:dmi:bvn*:bvr*:svnHP*:pnHPLaptop15s-eq0*:sku9MG38EA#ABZ:*
  KEYBOARD_KEY_9d=102nd                                  # Greater than/Less than
 
 # Elitebook
